docs/source/use_cases/privacy_protection_people_screens.rst

```diff
@@ -0,0 +1,128 @@
+
+Overview
+========
+
+Videos and Pictures often contain people and other sensitive visual information (e.g., the display
+on computer screens), even though this information might not be needed at all for visual
+processing. Our solution performs full body anonymization and computer screen blurring to protect
+the identities of individuals, and the sensitive information on computer screens from footage or
+pictures. It can be used to comply with the General Data Protection Regulation (GDPR) or other data
+privacy laws.
+
+.. image:: /assets/use_cases/privacy_protection_people_screens.gif
+   :class: no-scaled-link
+   :width: 50 %
+
+In this example use case, we want to count the number of people in the office, but we don't want
+compromise the privacy of the office inhabitants, nor the privacy of any information accidentally
+shown on the computer screens.
+
+Our solution automatically detects people, laptop and computer screens, and then blurs them. This is
+explained in the `How It Works`_ section.
+
+Demo
+====
+
+.. |pipeline_config| replace:: privacy_protection_people_screens.yml
+.. _pipeline_config: https://github.com/aimakerspace/PeekingDuck/blob/main/use_cases/privacy_protection_people_screens.yml
+
+To try our solution on your own computer, :doc:`install </getting_started/02_standard_install>` and run
+PeekingDuck with the configuration file |pipeline_config|_ as shown:
+
+.. admonition:: Terminal Session
+
+    | \ :blue:`[~user]` \ > \ :green:`peekingduck run -\-config_path <path/to/`\ |pipeline_config|\ :green:`>`
+
+How It Works
+============
+
+There are 2 main components to our solution:
+
+#. Person and computer screen segmentation, and
+#. Person and computer screen blurring.
+
+**1. Person and Computer Screen Segmentation**
+
+We use an open source instance segmentation model known as `Mask R-CNN <https://arxiv.org/abs/1703.06870>`_
+to obtain the masks of persons, computer screens and laptops. The masks are akin to the input frames
+or images, except that it only has a single channel and each pixel on the mask is a binary of either
+1 or 0, which indicates whether a specific class of thing is present (1) or absent (0) in a
+particular location of the image. For more information on how to adjust the ``mask_rcnn`` node, check
+out its :doc:`configurable parameters </nodes/model.mask_rcnn>`.
+
+**2. Person and Computer Screen Blurring**
+
+To blur the people and computer screens, we pixelate or gaussian blur the image pixels where the
+pixel values of the relevant masks are equal to 1 (indicating presence of object).
+
+Nodes Used
+==========
+
+These are the nodes used in the earlier demo (also in |pipeline_config|_):
+
+.. code-block:: yaml
+
+    nodes:
+    - input.visual:
+        source: <path/to/video>
+    - model.mask_rcnn:
+        detect: ["tv", "laptop"]
+    - draw.instance_mask:
+        effect: {blur: 50}
+    - model.mask_rcnn:
+        detect: ["person"]
+    - dabble.bbox_count
+    - draw.instance_mask:
```

Comment:
As discussed, we should explain why `model.mask_rcnn` and `draw.instance_mask` is repeated, and that it's not required if only anonymization is required.

Reply:
Footnote is added at the bottom in Italic